### DISCUSSION TITLE
Solved: How to connect a relay 4 powering a spindle?

### DISCUSSION THREAD

**Discussion by dasarne** [Expatria-Technologies/grblhal_2000_PrintNC#7] [General]
Hi, 
I bought a R5-grblHAL2000 board. Everything went smoothly, nice board.
Unfortunately, I did not manage to control a relay that can switch a spindle on/off. I use a simple milling cutter as a spindle, such as a Dremel.
Is there a tutorial on this topic somewhere? How can I connect a relay?

Yours
 Arne

**Comment by andrewmarles** [comment]
If your relay coil does not draw too much current (250 mA or so) the easiest way is to connect the jumper in the spindle section to the onboard 12V and take the positive lead of the spindle power connector to the + on your relay, and connect the negative to the spindle enable pin.  Clearly this would need a 12V relay.

**Comment by dasarne** [comment]
Hi Andrew,
thank you for the quick reply. You are one step ahead than me. Shouldn't there be 12V at P9 between A1 and A2 when I give a M3 command and 0V when I give a M5 command to the controller. Am I correct?
![2022-05-23 10_39_35-Window](https://user-images.githubusercontent.com/8106470/169780717-519ac9d9-a6e9-4ab3-92d2-41c8c01dfc3e.png)
Shouldn't D26 show the status of the spindle? Unfortunately, neither of these works. I saw that there should be a high signal coming from the Teensy at optocoupler U24 pin 1, but I don't see that either. 
![2022-05-23 10_42_27-Window](https://user-images.githubusercontent.com/8106470/169780832-f787dcaf-9c83-4367-b371-a651c2a09398.png)
![2022-05-23 10_45_55-PrintNC_GRBLHAL_Breakout_Schematic_A5_FULL_ASSEMBLY PDF - Adobe Acrobat Reader D](https://user-images.githubusercontent.com/8106470/169780908-e9fed849-934b-49c3-a98e-51d946c69f66.png)


What can be the cause? 
How do I proceed now?

**Comment by andrewmarles** [comment]
Place a jumper on pins 2-3 of P8 to connect the spindle section to onboard 12v.  P9 is not controlled by anything, it provides access to the power for the spindle section. If your teensy is not outputting the signal correctly that is likely a software or configuration issue.

**Comment by dasarne** [comment]
Thanks for the tip. That did the trick.
